src/main/java/core/basesyntax/Properties.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Properties implements AreaObtain, FigureDraw {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private String color = colorSupplier.getRandomColor();

```

Comment:
```suggestion
    private String color;

```
Add the constructor here